datahub-web-react/src/app/ingest/source/IngestionSourceList.tsx

```diff
@@ -173,14 +173,33 @@ export const IngestionSourceList = () => {
         setFocusSourceUrn(undefined);
     };
 
+    const sanitizeExtraArgs = (extraArgs): StringMapEntryInput[] => {
```

Comment:
maybe `formatExtraArgs` instead of "sanitize" - sanitize implies cleaning for security purposes like when we sanitize rich text before rendering it on the UI